extensions/ql-vscode/src/model-editor/model-alerts/model-alerts-view.ts

```diff
@@ -81,6 +81,12 @@ export class ModelAlertsView extends AbstractWebview<
       case "openModelPack":
         await this.app.commands.execute("revealInExplorer", Uri.file(msg.path));
         break;
+      case "openLogs":
+        await this.app.commands.execute(
+          "codeQL.openVariantAnalysisLogs",
+          123, // TODO: Get the actual variantAnalysisId
```

Comment:
I could figure out how to get the variant analysis (ID) at this point 🤔

Reply:
You can get send the id in the message from the view.